Src/WitsmlExplorer.Frontend/components/Modals/CopyMnemonicsModal.tsx

```diff
@@ -0,0 +1,234 @@
+      source: sourceReferences,
+      target: targetReference,
+      startIndex: startIndex,
+      endIndex: endIndex
+    };
+
+    if (sourceServer) {
+      AuthorizationService.setSourceServer(sourceServer);
+      JobService.orderJobAtServer(jobType, copyJob, targetServer, sourceServer);
+    } else {
+      await JobService.orderJob(jobType, copyJob);
+    }
+  }
+
+  async function orderDeleteInsertJob() {
+    const parentUid = targetReference.uid;
+    const parentType = getParentType(ComponentType.Mnemonic);
+
+    const targetParent = await ObjectService.getObject(
+      targetReference.wellUid,
+      targetReference.wellboreUid,
+      parentUid,
+      parentType,
+      undefined,
+      targetServer
+    );
+
+    const allTargetComponents = await ComponentService.getComponents(
+      targetReference.wellUid,
+      targetReference.wellboreUid,
+      targetReference.uid,
+      ComponentType.Mnemonic,
+      targetServer
+    );
+
+    const indexCurve = (targetParent as LogObject)?.indexCurve;
+
+    const targetComponentsToDelete = allTargetComponents.filter(
+      (c) =>
+        c.mnemonic !== indexCurve &&
+        sourceReferences.componentUids.find((sr) => sr === c.mnemonic)
+    );
+
+    if (targetComponentsToDelete.length == 0) {
+      await orderPasteJob();
+    } else {
+      const deleteJob: DeleteComponentsJob = {
+        toDelete: createComponentReferences(
+          targetComponentsToDelete.map((component) => component.mnemonic),
+          targetParent,
+          ComponentType.Mnemonic,
+          targetServer.url
+        )
+      };
+
+      const copyJob: CopyComponentsJob = {
+        source: sourceReferences,
+        target: targetReference
+      };
+
+      const replaceJob: ReplaceComponentsJob = { deleteJob, copyJob };
+
+      if (sourceServer) {
+        await JobService.orderJobAtServer(
+          JobType.ReplaceComponents,
+          replaceJob,
+          targetServer,
+          sourceServer
+        );
+      } else {
+        await JobService.orderJob(JobType.ReplaceComponents, replaceJob);
+      }
+    }
+  }
+
+  return (
+    <>
+      <ModalDialog
+        heading={`Paste`}
+        confirmText={`Submit`}
+        content={
+          <ContentLayout>
+            <TextLayout>
+              <Typography variant="h4">Choose paste option:</Typography>
+            </TextLayout>
+            <RadioLayout>
+              <RadioItemLayout>
+                <div style={{ alignItems: "top" }}>
+                  <Radio
+                    checked={
+                      selectedCopyMnemonicsType ===
+                      CopyMnemonicsType.DeleteInsert
+                    }
+                    onChange={() =>
+                      setCopyMnemonicsType(CopyMnemonicsType.DeleteInsert)
+                    }
+                  />
+                </div>
+                <div>
+                  <Typography variant="h5">Delete/Insert</Typography>
```

Comment:
Seems like the h-variants are not configured with the correct style. In `GlobalStyles.tsx` around line 130, I think we can remove the p so that it applies to all the typographies.

Reply:
I've removed the _p_. However, I am not sure I see much of a difference. Hopefully nothing's broken.